pkg/netutils/interface.go

```diff
@@ -7,10 +7,13 @@ import (
 	"io"
 	"net"
 	"os"
-
 	"strings"
 
+	"github.com/alibaba/pouch/pkg/exec"
+	"github.com/sirupsen/logrus"
```

Comment:
I am not sure if we need to import another pkg package in this pkg package.
But I am afraid there is no need to import the package `logrus`.
We do not want to record an infof log in a common pkg package.

Reply:
> We do not want to record an infof log in a common pkg package.

I agree with you.